docs/components/sensors/temperature-sensor.md

```diff
@@ -1,3 +1,40 @@
-# Temperature Sensor
 
-{!includes/sip.md!}
+# Temperature
+
+## Overview
+  
+A non-visible Component that measures the temperature of air surrounding the device in unit of Degree Celsius.
+
+  
+## Properties
+
+  
+
+### Enabled 
+[ Getter + Setter ] 
+Type: Boolean    
+
+![image](url) 
+
+If checked the sensor will be enabled
+
+  
+### Available 
+[ Getter Only ]  
+Type: Boolean  
+![image](url)   
+
+Check if the device has Temperature sensor.  
+
+
+### Temperature 
+[ Getter Only ]  
+Type: Number   
+![image](url)   
+
+Return the temperature in degree Celsius.
+## Events  
+### Temperature Changed  
+[ Number `Tempereature` ]   
+![image](url)  
+Event that fires when temperature is changed. 
```

Comment:
Please use standard sentence structures, and a table for event parameters. Use screenshot.md as reference

Reply:
fixed